sox.features.info.json

```diff
@@ -251,45 +252,46 @@
             "name": "editComment",
             "desc": "Pre-defined edit comment options (checkboxes)",
             "extended_description": "Adds checkboxes to add canned messages for edit revisions when editing. You can make *your own* canned responses by clicking the 'Edit Reasons' button that is added to the Help dropdown menu in the topbar",
-            "meta": "http://meta.stackexchange.com/questions/190461/improve-the-editing-flow-with-predefined-options-for-edit-summary",
+            "meta": "https://meta.stackexchange.com/q/190461",
             "match": "*://*/questions/*",
-            "exclude": "*://chat.*.com/*,SE1.0,*://*/questions/ask",
+            "exclude": "SE1.0,*://*/questions/ask,*://*/questions/tagged*,*://*/questions/greatest-hits*",
             "feature_packs": ["power_user"]
         }, {
             "name": "editReasonTooltip",
             "desc": "Add a tooltip to posts showing the last revision's comment",
             "extended_description": "When a post is edited, the editor is displayed underneath the post. This feature will show what the last revision's comment was as a tooltip when you hover over 'edited' underneath a post in 'edited [date] at [time]'",
-            "meta": "http://meta.stackexchange.com/questions/2315/show-reason-for-edit-without-clicking-through-to-diff",
-            "match": "*://*/questions*,*://*/review*",
-            "exclude": "SE1.0,*://*/questions/ask",
-            "feature_packs": ["power_user"]
+            "meta": "https://meta.stackexchange.com/q/2315",
+            "match": "*://*/questions/*,*://*/review*",
+            "exclude": "SE1.0,*://*/questions/ask,*://*/questions/tagged*,*://*/questions/greatest-hits*",
+            "feature_packs": ["power_user"],
+            "usesApi": true
         }, {
             "name": "findAndReplace",
             "desc": "Add a find & replace feature to the markdown editor",
             "extended_description": "",
             "meta": "",
-            "match": "*://*/questions*,*://*/review*",
-            "exclude": "SE1.0"
+            "match": "*://*/questions/*,*://*/review*",
+            "exclude": "SE1.0,*://*/questions/ask,*://*/questions/tagged*,*://*/questions/greatest-hits*"
```

Comment:
I think this feature should work on the `/ask` page because it has a markdown editor!